package.json

```diff
@@ -38,18 +38,24 @@
     "ethers": "^6.13.2",
     "google-protobuf": "^3.21.4",
     "id128": "^1.6.6",
-    "lodash": "^4.17.21"
+    "lodash": "^4.17.21",
+    "ulid": "^2.3.0"
```

Comment:
The ulid is for id generation, which is mainly used in test cases. Is `"id128": "^1.6.6",` sufficient, that we don’t need `ulid`?

Reply:
we can use anything. no need ulid or uuid. we don't enforce constraint on the id of the node. it's just a string. as long as it's unique among the node, that's enough to identified it.

feel free to remove ulid.

Reply:
Remove the `ulid` library and rely on id128 soley, since it can provide sortable ulid.